caffe2/python/operator_test/conv_test.py

```diff
@@ -219,13 +259,27 @@ def test_convolution_gradients(self, op_type, stride, pad, kernel, dilation,
                 self.assertDeviceChecks(dc, op, inputs, [0])
             return
 
-        self.assertDeviceChecks(dc, op, inputs, [0])
+        try:
+            self.assertDeviceChecks(dc, op, inputs, [0])
+        except RuntimeError as e:
+            es = str(e)
+            if es.find("status == CUDNN_STATUS_SUCCESS") < 0 \
```

Comment:
nit: more pythonic way would be `if xxx not in es`

Reply:
ok.